visualizer/index.html

```diff
@@ -46,6 +46,7 @@
     }
     #visualizerBody {
       flex-direction: column;
+      overflow: auto;
```

Comment:
I don't think you'll need this anymore. As I was looking at this pull request (and trying to see why you needed to add the `overflow` here), I realized that the addition of the `#visualizerBody` in your earlier PR caused some problems with the scrolling. I've just pushed a fix, so I think you can remove this line from your patch now.